packages/block-library/src/buttons/test/__snapshots__/edit.native.js.snap

```diff
@@ -71,3 +71,60 @@ exports[`Buttons block when a button is shown removing button along with buttons
 <p></p>
 <!-- /wp:paragraph -->"
 `;
+
+exports[`Buttons block when a button is shown removing button along with buttons block removes the button and buttons block when deleting the block using the delete (backspace) key 1`] = `
+"<!-- wp:buttons -->
+<div class="wp-block-buttons"><!-- wp:button /--></div>
+<!-- /wp:buttons -->"
+`;
```

Comment:
> removes the button and buttons block when deleting the block using the delete (backspace) key

Not sure if the snapshot reflects the correct state. If the Buttons block is removed when using the backspace key, I understand this should be empty, is it?

Reply:
Yes, I agree. Perhaps the backspace `fireEvent` is not fully removing the block within the test. 🤔